evap/student/templates/student_vote.html

```diff
@@ -82,10 +79,7 @@ <h3 class="panel-title">{{ contributor.full_name }}</h3>
                                         <table class="table table-striped">
                                             <tbody>
                                                 {% for field in form %}
-                                                    <tr class="vote-row">
-                                                        <td>{{ field.label }}</td>
-                                                        <td class="vote-inputs {% if preview %}preview{% endif %}">{{ field.errors }}{{ field }}</td>
-                                                    </tr>
+                                                    {% include "student_vote_row.html" %}
```

Comment:
you need to create a inclusion tag for that. including templates without such a tag (and without the template cache) costs a lot of performance if doing it multiple times on one page.